app/containers/Commit.tsx

```diff
@@ -1,8 +1,56 @@
 import * as React from "react";
+import { Col, Row } from "react-bootstrap";
+import { CommitFundsDescription } from "../components/CommitFundsDescription";
+import CommitFundsForm from "../components/CommitFundsForm";
 import { CommitNavbar } from "../components/CommitNavbar";
+import { UserAddressComponent } from "../components/UserAddressComponent";
+import * as style from "./Commit.scss";
 
 export const Commit: React.SFC = () => {
-  return <CommitNavbar>Commit funds in Neufund Commitment Opportunity</CommitNavbar>;
+  return (
+    <div className={style.pageContainer}>
+      <CommitNavbar>Commit funds in Neufund Commitment Opportunity</CommitNavbar>
+      <Row>
+        <Col sm={10} smOffset={1} className={style.contentContainer}>
+          <Row>
+            <Col sm={12}>
+              <h2 className={style.sectionHeader}>
```

Comment:
what about extracting it to separe component? Like <TitleHeader number={1}>Commit funds</TitleHeader>

Reply:
fixed